tests/test

```diff
@@ -24,7 +24,7 @@ if [ $(basename $(sudo readlink /proc/1/exe)) == systemd ] ; then
 fi
 
 STAGE1_SRC_FROM_HOST=0
-if tar tvf ../bin/stage1.aci rootfs/flavor|grep -q usr-from-host ; then
+if tar xOvf ../bin/stage1.aci manifest|grep -q usr-from-host ; then
```

Comment:
not sure about the fonts, it should be 'capital oh' rather than 'digit zero'. hopefully the markdown formatted example below should be copy&pastable into terminal and working correctly:

```
cd `mktemp -d`
echo foo > bar
tar czf foo.tgz bar
rm bar
tar xvOf foo.tgz bar
```